src/Raven.Server/Documents/ETL/Providers/Queue/BlittableJsonEventBinaryFormatter.cs

```diff
@@ -28,7 +30,7 @@ public override ReadOnlyMemory<byte> EncodeBinaryModeEventData(CloudEvent cloudE
                 return Array.Empty<byte>();
             }
 
-            MemoryStream ms = _ctx.CheckoutMemoryStream();
+            var ms = new MemoryStream();
```

Comment:
This will be called for every _queue_ ETL message. The usage of cached memory stream was quite important this this case. Is there any reason we don't use `RecyclableStream` here?

Reply:
Ups, thx!